stdlib/builtin/nil_class.rbs

```diff
@@ -32,6 +32,13 @@ class NilClass < Object
   # ```
   def to_h: () -> ::Hash[untyped, untyped]
 
+  # Always returns zero.
+  # 
+  # ```ruby
+  # nil.to_i   #=> 0.0
+  # ```
+  def to_i: () -> Integer
```

Comment:
`nil.to_i` always returns 0, IMO literal type `0` is better than Integer

Reply:
Thank you for the review. I didn't know there was a literal type.
Fixed https://github.com/ruby/ruby-signature/pull/66/commits/cdfe4e5709b3d58ec7c0905eec18cc55bf6c44ee